admin/src/locales/de.json

```diff
@@ -1,4 +1,65 @@
 {
+  "bookmark": "bookmark",
+  "creation_form": {
+    "creation_for": "Schöpfung für ",
+    "enter_text": "Text eintragen",
+    "form": "Creation form",
+    "min_characters": "Mindestens 10 Zeichen eingeben",
+    "reset": "zurücksetzen",
+    "select_month": "Monat Auswählen",
+    "select_value": "Betrag Auswählen",
+    "submit_creation": "Schöpfung einreichen",
+    "toasted": "Offene Schöpfung ({value} GDD) für {email} wurde gespeichert und liegen zur Bestätigung bereit",
+    "toasted_update": "`Offene schöpfung {value} GDD) für {email} wurde geändert, liegt zur Bestätigung bereit",
+    "update_creation": "Update Schöpfung"
+  },
+  "details": "Details",
+  "e_mail": "E-Mail",
+  "firstname": "Vorname",
+  "gradido_admin_footer": "Gradido Akademie Adminkonsole",
+  "hide_details": "Details verbergen von ",
```

Comment:
Das Leerzeichen am Ende sollte besser in der Komponente sein

Reply:
erledigt